src-script/build-release-package.js

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env node
```

Comment:
Just for my own recollection. Forgot what this was. Why do we do this?

Reply:
#!/usr/bin/env node is an instance of a [shebang line](https://en.wikipedia.org/wiki/Shebang_(Unix)): the very first line in an executable plain-text file on Unix-like platforms that tells the system what interpreter to pass that file to for execution, via the command line following the magic #! prefix (called shebang).

From stack overflow^